b2sdk/raw_api.py

```diff
@@ -361,6 +361,8 @@ def get_upload_file_headers(
             'Content-Type': content_type,
             'X-Bz-Content-Sha1': content_sha1,
         }
+        if cache_control is not None:
```

Comment:
I left if here to avoid backwards incompatible changes in `B2RawHTTPApi` because I didn't realize it's considered internal despite the fact that it's exported in `vX` modules. But since it is, I'll just get rid of it.